rpc/cfx_api.go

```diff
@@ -115,10 +116,10 @@ func (api *cfxAPI) GetCode(ctx context.Context, contract types.Address, epoch *t
 	return cfx.GetCode(contract, toSlice(epoch)...)
 }
 
-func (api *cfxAPI) GetStorageAt(ctx context.Context, address types.Address, position *hexutil.Big, epoch *types.Epoch) (hexutil.Bytes, error) {
+func (api *cfxAPI) GetStorageAt(ctx context.Context, address types.Address, location *rpcgtypes.Location, epoch *types.Epoch) (hexutil.Bytes, error) {
```

Comment:
This seems to be an overkill. Geth uses `common.Hash` and works well with both types of calls. Can we just use `common.Hash`?

Reply:
`common.Hash` checks during unmarhsalling that the input is of length exactly 32. And I tested it locally, it works this way. I didn't find any time that is suitable here

Reply:
Okay, maybe this is the entry point for geth's implementation then, can we use the same approach?

https://github.com/scroll-tech/go-ethereum/blob/a8a9dc83cb697f5ecb764acab8ef93ffa6f66379/internal/ethapi/api.go#L833-L843

Reply:
yes, nice point, changed to string